BlazorServer/Pages/_Host.cshtml

```diff
@@ -8,28 +8,29 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>BlazorServer</title>
-    <base href="~/" />
-    <link rel="stylesheet" href="css/bootstrap/bootstrap.icons.min.css" />
-    <link rel="stylesheet" href="css/bootstrap/bootstrap-theme.min.css" />
-    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
-    @*<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.0/sketchy/bootstrap.min.css" />*@
-
-    <script src="_content/MatBlazor/dist/matBlazor.js"></script>
-    <link href="_content/MatBlazor/dist/matBlazor.css" rel="stylesheet" />
-
-    <link href="css/site.css" rel="stylesheet" />
-    <script src="https://d3js.org/d3.v4.min.js"></script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" />
-
-    <script>const whTooltips = { colorLinks: true, iconizeLinks: true, renameLinks: true };</script>
-    <script src="https://wow.zamimg.com/widgets/power.js"></script>
-
-    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-    <script src="https://www.youtube.com/iframe_api"></script>
-    <script src="~/script/script.js"></script>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>BlazorServer</title>
+  <base href="~/" />
+  <link rel="stylesheet" href="css/bootstrap/bootstrap.icons.min.css" />
+  <link rel="stylesheet" href="css/bootstrap/bootstrap-theme.min.css" />
+  <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+  <link href="_content/Syncfusion.Blazor/styles/bootstrap-dark.css" rel="stylesheet" />
+  @*<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.0/sketchy/bootstrap.min.css" />*@
+
+  <script src="_content/MatBlazor/dist/matBlazor.js"></script>
+  <link href="_content/MatBlazor/dist/matBlazor.css" rel="stylesheet" />
+
+  <link href="css/site.css" rel="stylesheet" />
+  <script src="https://d3js.org/d3.v4.min.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" />
+
+  <script>const whTooltips = { colorLinks: true, iconizeLinks: true, renameLinks: true };</script>
+  <script src="https://wow.zamimg.com/widgets/power.js"></script>
+
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="https://www.youtube.com/iframe_api"></script>
+  <script src="~/script/script.js"></script>
```

Comment:
if I'm not mistaken, only the indentation changed and `_content/Syncfusion.Blazor/styles/bootstrap-dark.css` added

Reply:
yes you're right mate